saleor/plugins/webhook/tests/test_webhook.py

```diff
@@ -147,6 +147,18 @@ def test_product_updated(mocked_webhook_trigger, settings, product):
     )
 
 
+@mock.patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_product_deleted(mocked_webhook_trigger, settings, product):
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+    manager = get_plugins_manager()
+    manager.product_deleted(product)
+
+    expected_data = generate_product_payload(product)
```

Comment:
Can we delete the product first? Then assign ID once again and generate the payload?
We should simulate that product was deleted for real